blocks/gmo-program-details/gmo-program-details.js

```diff
@@ -149,23 +173,29 @@ export default async function decorate(block) {
     </div>
     `;
     buildProductCard(program);
-    try {
-        const imageObject = await searchAsset(program.programName, program.campaignName);
-        if (imageObject){
-          insertImageIntoCampaignImg(block,imageObject);
-          document.getElementById('totalassets').textContent = imageObject.assetCount;
-        }
-        else
-        {
-          document.getElementById('totalassets').textContent = 0;
-        }
-    } catch (error) {
-        console.error("Failed to load campaign image:", error);
+
+    if (imageObject){
```

Comment:
looks like this code is duplicate and already have it in line 50. Do you need` if (program) {` condition in line number 47

Reply:
It is supposed to be like this.
At first the image and program details is displayed in the block without the deliverables.

Then I write the block again with the  program data, image and deliverable info